src/API/Google/MerchantPriceBenchmarks.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Google;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query\MerchantPriceBenchmarksQuery;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query\MerchantPriceSuggestionsQuery;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\OptionsAwareInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\OptionsAwareTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\ProductHelper;
+use Automattic\WooCommerce\GoogleListingsAndAds\Vendor\Google\Service\ShoppingContent;
+use Exception;
+
+/**
+ * Class MerchantPriceBenchmarks
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Google
+ */
+class MerchantPriceBenchmarks implements OptionsAwareInterface {
+
+	use OptionsAwareTrait;
+
+	/**
+	 * The shopping service.
+	 *
+	 * @var ShoppingContent
+	 */
+	protected $service;
+
+	/**
+	 * Product helper class.
+	 *
+	 * @var ProductHelper
+	 */
+	protected $product_helper;
```

Comment:
This isn't being used yet, but we may need to use it to convert the IDs returned from the API, similar to https://github.com/woocommerce/google-listings-and-ads/blob/925c3fb030447cdff4c10c0901c91a2ac300a282/src/API/Google/MerchantReport.php#L94